common/protob/messages-cardano.proto

```diff
@@ -50,22 +51,21 @@ message CardanoPublicKey {
  * Request: Ask device to sign Cardano transaction
  * @start
  * @next CardanoSignedTx
- * @next CardanoTxRequest
  * @next Failure
  */
 message CardanoSignTx {
     repeated CardanoTxInputType inputs = 1;     // inputs to be used in transaction
     repeated CardanoTxOutputType outputs = 2;   // outputs to be used in transaction
-    optional uint32 transactions_count = 3;     // transactions count
-    optional uint32 protocol_magic = 5;         // network's protocol magic
+    optional uint32 protocol_magic = 3;         // network's protocol magic
+    optional uint64 fee = 4;                    // transaction fee - added in shelley
+    optional uint64 ttl = 5;                    // transaction ttl - added in shelley
     /**
      * Structure representing cardano transaction input
      */
     message CardanoTxInputType {
         repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
         optional bytes prev_hash = 2;   // hash of previous transaction output to spend by this input
         optional uint32 prev_index = 3; // index of previous output to spend
-        optional uint32 type = 4;       // input type, defaults to 0
     }
```

Comment:
given the suggestion above, I would just comment out but not remove this so future contributors do not reuse this id